src/Test.UnitTests.Sarif/Visitors/GitHubDspIngestionVisitorTests.cs

```diff
@@ -42,18 +42,10 @@ public void GitHubDspIngestionVisitor_LimitsNumberOfResults()
             }
         }
 
-        [Fact]
-        public void GitHubDspIngestionVisitor_RemovesInvocations()
```

Comment:
>public [](start = 8, length = 6)

Before removing these tests, I first ensured that they passed with appropriate adjustments to the test data files. Having done that, IMO there's no point in keeping tests just to show that we _don't_ do something any more.